x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp/translations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
```

Comment:
I've moved an inline translation to this file, and created a format function for another. I decided to keep the one in `NoImageAvailable` there because I feel the usage of `FormattedMessage` improves readability, vs. importing. https://github.com/elastic/kibana/pull/89179/commits/ad1ebbd073d78b0ad3c27f47c165c06bdfca3aa8